kernprof.py

```diff
@@ -155,53 +155,46 @@ def find_script(script_name):
     raise SystemExit(1)
 
 
-def main(args=None):
-    def strictly_positive(option, opt, value, parser):
-        if value <= 0:
-            raise optparse.OptionValueError("option %s: floating-point value must be > 0, got %s" % (opt, value))
-        setattr(parser.values, option.dest, value)
-
-    if args is None:
-        args = sys.argv
-    usage = "%prog [-s setupfile] [-o output_file_path] scriptfile [arg] ..."
-    parser = optparse.OptionParser(usage=usage, version=__version__)
-    parser.allow_interspersed_args = False
-    parser.add_option('-l', '--line-by-line', action='store_true',
-        help="Use the line-by-line profiler from the line_profiler module "
-        "instead of Profile. Implies --builtin.")
-    parser.add_option('-b', '--builtin', action='store_true',
-        help="Put 'profile' in the builtins. Use 'profile.enable()' and "
-            "'profile.disable()' in your code to turn it on and off, or "
-            "'@profile' to decorate a single function, or 'with profile:' "
-            "to profile a single section of code.")
-    parser.add_option('-o', '--outfile', default=None,
-        help="Save stats to <outfile>")
-    parser.add_option('-s', '--setup', default=None,
+def main():
```

Comment:
I'd like to keep the `args` signature if possible.

Reply:
Done, cf my comment in the other thread